### DISCUSSION TITLE
Welcome to SFC-foundations Discussions!

### DISCUSSION THREAD

**Discussion by himanshu007-creator** [himanshu007-creator/SFC-foundations#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this fellow developers.

**Comment by aarushi31** [comment]
Hi I'm a GSSoC'21 participant. I would like to contribute to this project so could I get added to the discord server for the same?

**Comment by sudeepshukla930** [comment]
Hey  @himanshu007-creator , @divyashukla23  ,   I was just viewing the website , I found that :
1. There is the lack of hover effect in header
2.  Little bit of enhancement in style 
3.  header is long .

what your view on it .
Waiting for your response @himanshu007-creator .
